src/ontology/ro-edit.owl

```diff
@@ -6564,15 +6568,14 @@ IrreflexiveObjectProperty(obo:RO_0017004)
 
 # Object Property: obo:RO_0017005 (contains measured amount)
 
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0017005 "anticoagulant-containing test tube contains measured amount 5 ml of blood specimen.")
 AnnotationAssertion(obo:IAO_0000115 obo:RO_0017005 "A relation between a container and measurement datum that specifies the actual amount of material in the container.")
 AnnotationAssertion(terms:contributor obo:RO_0017005 <https://orcid.org/0000-0002-2999-0103>)
 AnnotationAssertion(terms:contributor obo:RO_0017005 <https://orcid.org/0000-0002-5714-991X>)
-AnnotationAssertion(obo:IAO_0000112 obo:RO_0017005 "anticoagulant-containing test tube contains measured amount 5 ml of blood specimen.")
```

Comment:
@anitacaron do you know why these spurious diffs still exist?

Reply:
That's fine. The serialization here sometimes is an issue.